diff --git a/lib/main.dart b/lib/main.dart
index 19d079b..486730e 100644
--- a/lib/main.dart
+++ b/lib/main.dart
@@ -19,6 +19,8 @@ import 'package:Openbook/services/localization.dart';
 import 'package:Openbook/services/universal_links/universal_links.dart';
 import 'package:Openbook/widgets/toast.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/foundation.dart'
+    show debugDefaultTargetPlatformOverride;
 import 'package:flutter\_localizations/flutter\_localizations.dart';
 import 'package:sentry/sentry.dart';
 import 'dart:async';
@@ -137,6 +139,7 @@ class MyApp extends StatelessWidget {
 }
 
 Future<Null> main() async {
+  debugDefaultTargetPlatformOverride = TargetPlatform.android;
   // This captures errors reported by the Flutter framework.
   FlutterError.onError = (FlutterErrorDetails details) async {
     if (isInDebugMode) {
